OptimizelySDKCore/OptimizelySDKCore/OPTLYDecisionService.m

```diff
@@ -487,14 +487,17 @@ - (BOOL)evaluateAudienceConditionsForExperiment:(OPTLYExperiment *)experiment
     // Log Experiment Evaluation Started
     NSString *logMessage = [NSString stringWithFormat:OPTLYLoggerMessagesAudienceEvaluatorEvaluationStartedForExperiment, experiment.experimentKey, [experiment getAudienceConditionsString]];
     [config.logger logMessage:logMessage withLevel:OptimizelyLogLevelDebug];
-    BOOL areAttributesValid = [[experiment evaluateConditionsWithAttributes:attributes projectConfig:config] boolValue];
```

Comment:
Actually, you know what, I missed the `... boolValue]` bit before, and totally forgot that we only want to log `@"TRUE"`/`@"FALSE"` for an experiment's audience _combinations_.

So the version of this file on the left-hand side might be better after all! Sorry for making you go back and forth. 😬

The only thing is to make sure we can call `... boolValue]` even when `evaluateConditionsWithAttributes` returns null. I'm not familiar with Objective-C, but that _seems_ likely to produce a null pointer error. I don't care how we handle that case, as long we log `@"FALSE"` and return `false`, with a unit test to make sure.